portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -51,13 +54,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get the input from the form.
     String text = getParameter(request, "text-input", "");
 
-    // Break the text into individual words.
-    String[] words = text.split("\\s*,\\s*");
+    //create entity
+    Entity taskEntity = new Entity("Task");
+    taskEntity.setProperty("title", text);
 
-    // Respond with the result.
-    response.setContentType("text/html;");
+    //"Store comments"
     myComments.add(text);
-    response.getWriter().println(Arrays.toString(words));
+    
+    //store entity
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.put(taskEntity);
 
     // Redirect back to the HTML page.
     response.sendRedirect("/index.html");
```

Comment:
Would you add some changes to index.html to view the added comments?

Reply:
I'm not exactly sure what you mean?

Reply:
Hi Dese, after adding the comments the page is redirected to index.html page, right?I did not see any changes for listing out these comments for this pull release. It is fine if you do it for the next planned release. You might just want to leave that in the pull request description so that reviewer knows.

Reply:
Oh if you press the button the comments show up. Was it meant to show up on its own on the page when the page was reloaded? If so I'll get on that. That was one of the things I was confused about when writing code for that bit